tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -191,7 +191,7 @@ public function testEventManagerPassedToPlatform()
     public function testDriverExceptionIsWrapped($method)
     {
         $this->expectException(DBALException::class);
-        $this->expectExceptionMessage("An exception occurred while executing 'MUUHAAAAHAAAA':\n\nSQLSTATE[HY000]: General error: 1 near \"MUUHAAAAHAAAA\"");
+        $this->expectExceptionMessage("An exception occurred while executing \"MUUHAAAAHAAAA\":\n\nSQLSTATE[HY000]: General error: 1 near \"MUUHAAAAHAAAA\"");
```

Comment:
`phpcbf` should have converted the outer quotes to `'`. Not sure why it didn't.

Reply:
Maybe because it has `\n\n` inside?

Reply:
Indeed.